bin/varnishd/cache/cache_vrt.c

```diff
@@ -124,8 +124,9 @@ VRT_hit_for_pass(VRT_CTX, VCL_DURATION d)
 	oc->ttl = d;
 	oc->grace = 0.0;
 	oc->keep = 0.0;
-	VSLb(ctx->vsl, SLT_TTL, "HFP %.0f %.0f %.0f %.0f uncacheable",
-	    oc->ttl, oc->grace, oc->keep, oc->t_origin);
+	if (!(oc->flags & OC_F_PRIVATE))
```

Comment:
so this is a `return(pass())` for a `bereq.uncacheable` object.

I wonder if it would rather be more stringent to log the error as in L119, but maybe with some reason added (called from the client side vs. called on an already uncacheable object).
And I wonder if that should rather be an `SLT_Notice`?